test/forge/child/validator/RewardPool.t.sol

```diff
@@ -17,12 +17,13 @@ abstract contract Uninitialized is Test {
     RewardPool pool;
     address rewardWallet = makeAddr("rewardWallet");
     address alice = makeAddr("alice");
+    uint256 epochSize = 64;
```

Comment:
how clear is it that this will never change? I'm wondering if it needs a setter function

Reply:
Could you clarify?

There is a setter in `NetworkParams` which also emits an event. `epochSize` is then passed on reward distribution. There are additional checks for `epochSize` in the function.

The value hardcoded in the tests is for convenience.

Reply:
silly mistake on my part